### PR TITLE
Adds some sound for cryo pods, suit storage, holopads, and fire alarms

### PR DESCRIPTION
**Pull request by ghost**
:cl: coldud13, Qustinnus
 sound: New audio for cryo pods, suit storage, holopads, and fire alarms
/:cl:

some samples:

https://streamable.com/cjlsyd

https://streamable.com/gn8r6a

this needs TM'ing because this does something experimental with sound loops.

credits to https://github.com/Lumos-SS13/Lumos/pull/118 for the audio

### CODE REVIEW

**Comment by Qustinnus** [comment]
all my homies hate the broken changelog shit

**Comment by Jared-Fogle** [comment]
cryo should be bubbly

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -348,6 +348,8 @@
 */
 /obj/machinery/suit_storage_unit/proc/cook()
 	var/mob/living/mob_occupant = occupant
+	if(!uv)
+		playsound(src, 'sound/machines/flushrad.ogg', 50, 0)
 	if(uv_cycles)
```
Comment (ShizCalev) [inline code/game/machinery/suit_storage_unit.dm:353]: why not just have that under here?

**Comment by Qustinnus** [comment]
I'll re-open this when i re-worked them

**Pull request closed without merging**